core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -349,26 +349,25 @@ public Configuration defineConfiguration(String name, String template, Configura
          if (c == null) {
             throw log.undeclaredConfiguration(template, name);
          } else {
-            return defineConfiguration(name, false, configurationOverride, c);
+            return actualDefineConfiguration(name, configurationOverride, c);
          }
       }
-      return defineConfiguration(name, configurationOverride);
+      return actualDefineConfiguration(name, configurationOverride);
    }
 
-   private Configuration defineConfiguration(String name, boolean checkExisting, Configuration... configurations) {
+   private Configuration actualDefineConfiguration(String name, Configuration... configurations) {
```

Comment:
Nitpicking: I'd prefer `internalDefineConfiguration` or `doDefineConfiguration`, `actual` is an adjective so it fits better when delegating to another object.